### PR TITLE
Bump pgi requirement to 0.0.11.2.

### PR DESCRIPTION
**Pull request by anntzer**
This version of pgi fixes a bug in the calling of callbacks, which
affected timers in the GTK3 backends.  Remove a special-casing in the
test suite accordingly (in test_backends_interactive.py).

(See https://github.com/pygobject/pgi/pull/48.)

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

**Comment by anntzer** [comment]
Now it's working...

File: lib/matplotlib/tests/test_backends_interactive.py [hunk 1]
```diff
@@ -34,16 +35,15 @@ def _get_testable_interactive_backends():
...
-#    key_press_event("q") from draw_event causes breakage due to the canvas
-#    widget being deleted too early.
-# 2. On gtk3, we cannot even test the timer setup (on Travis, which uses pgi)
-#    due to https://github.com/pygobject/pgi/issues/45.  So we just cleanly
-#    exit from the draw_event.
+# Using a timer not only allows testing of timers (on other backends), but is
+# also necessary on gtk3 and wx, where a direct call to key_press_event("q")
+# from draw_event causes breakage due to the canvas widget being deleted too
+# early.  Also, gtk3 redefines key_press_event with a different signature, so
+# we directly invoke it from the superclass instead.
```
Comment (anntzer) [inline lib/matplotlib/tests/test_backends_interactive.py]: xref https://github.com/matplotlib/matplotlib/issues/10836

**Comment by tacaswell** [comment]
attn @sandrotosi Is this going to be a problem for debian?

**Comment by anntzer** [comment]
I don't actually recommend anyone to use pgi (nor does the pgi maintainer); the only real use case is to be installable on Travis with dockerizing the entire build.

**Comment by sandrotosi** [comment]
thanks @tacaswell for pinging me! from the project description pgi is 'GTK+ / GObject Introspection Bindings for PyPy' - in debian we currently dont provide pypy packages for mpl, and we already have cairocffi in our dependencies, so i think we should be good here

**Pull request merged**